src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.tsx

```diff
@@ -2,26 +2,64 @@
 import { jsx } from '@emotion/react';
 import { PortraitVideoPromoProps } from '#app/models/types/portraitVideo';
 import Image from '#app/components/Image';
-import Heading from '../../Heading';
+import Text from '#app/components/Text';
+import { Play } from '#app/components/icons';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+import moment from 'moment';
+import formatDuration from '#app/lib/utilities/formatDuration';
+import { useContext } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
 import styles from './index.styles';
 
 export default (item: PortraitVideoPromoProps) => {
-  const { images, headlines, onClick } = item;
+  const { images, headlines, video, onClick } = item;
+  const { translations } = useContext(ServiceContext);
 
   const imageUrl = images?.[0]?.url;
   const alt = images?.[0]?.altText || '';
   const headline = headlines?.promoHeadline || '';
+  const mediaISO8601Duration = video?.version.duration;
+  const mediaType = 'video';
+
+  const durationTranslation = translations?.media?.duration || 'Duration';
+  let durationString = '';
+  let durationSpokenString = '';
+  if (mediaISO8601Duration) {
+    const separator = ':';
+    const momentDuration = moment.duration(mediaISO8601Duration, 'seconds');
+    durationString = formatDuration({
+      duration: momentDuration,
+      padMinutes: true,
+    });
+    durationSpokenString = formatDuration({
+      duration: momentDuration,
+      separator,
+    });
+  }
+
+  const hiddenText = `${headline}, ${mediaType}, ${mediaISO8601Duration ? `${durationTranslation}  ${durationSpokenString}, ` : ''}Play ${mediaType}`;
```

Comment:
Should we try to build in the translation mechanism for `Play` now, so that we don't have to come back and revisit it?

Reply:
Also, is it Play or is it Watch - because it will always be a video? There's another ticket being worked on for Live Media on Live Pages, and there does seem to be a translation for `watch` - see https://github.com/bbc/simorgh/pull/12758 - could we have a look?